app/backend/migrations/20220311142406-add-workcpace.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Adds sample workspace
+ */
+module.exports = {
+  async up(db) {
+    // Create a sample workspace named 'CodeX Tasks'
+    await db.collection('workspaces').insertOne({ name: 'CodeX Tasks' });
```

Comment:
Sorry, I could not understand why do we have the `workspaces` collection. And why do we have `workspaceId`?

We have no workspaces by design. The Task Manager is the single workspace itself. There is no ability to add/switch workspaces. And there is no need to have such a property as `workspaceId`.

We just have the workspace title and team settings. 

<img width="778" alt="image" src="https://user-images.githubusercontent.com/3684889/161564403-d09d196e-527d-4010-9f73-1e2f8b692d53.png">

Reply:
I thought the switching and adding workspace is going to be added in the future. If that is not, i'll just remove it.

Reply:
Maybe we could store workspace settings for example at the `workspace` collection instead of `workspaces.<_id>` ? What do you think?

Reply:
Not sure I understand it right. `Project` and `Teammate` should not have `workspaceId` or `workspace` properties, because there is only one workspace.